nautobot/core/utils/lookup.py

```diff
@@ -293,7 +293,7 @@ def get_table_class_string_from_view_name(view_name):
     view_class = getattr(view_func, "cls", getattr(view_func, "view_class", None))
     if hasattr(view_class, "table_class"):
```

Comment:
Can you add a similar check here as well?
```suggestion
    if hasattr(view_class, "table_class") and view_class.table_class:
```